app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,16 @@
+class ProjectsController < ApplicationController
+
+  def index
+    @projects = Project.all
+  end
+
+  def show
+    @project = Project.find(params[:id])
+  end
+
+  def create
+    @project = Project.find(params[:id])
+    @contestant = 
+    ContestantProject.create(contestant_id: @contestant.id, project_id: @project_1.id)
```

Comment:
No need to find the project again if you already have the params from the form URL.